gateway/1.4/modules/ROOT/pages/flex-conn-reg-run-app.adoc

```diff
@@ -5,31 +5,46 @@ include::_attributes.adoc[]
-|xref:flex-conn-reg-run-app.adoc#register-and-run-with-a-connected-app-as-a-kubernetes-ingress-controller[Register and Run With a Connected App as a Kubernetes Ingress Controller]
+|<<linux>>
+|<<docker>>
+|<<kubernetes>>
 |===
 
 // Prerequisites
 include::partial$task-reg-run-flex-gateway.adoc[tags=prerequisites-heading;app-prerequisites;prerequisites;environment-prerequisites]
 
+[[linux]]
 // Register and run with a connected app as a Linux service
 include::partial$task-reg-run-flex-gateway.adoc[tags=app-linux-heading;reg-command-intro;app-replace-content;environment-replace-content;replace-content;reg-command-heading;reg-command-1;linux-reg-command;app-reg-command;environment-reg-command;connected-reg-command;organization-reg-command;output-reg-command-linux;reg-command-2;after-reg;linux-after-reg;connected-after-reg;create-config-folder-file;config-content;linux-start-commands;gateway-connected]
 
+[[docker]]
 // Register and run with a connected app in a Docker container
 include::partial$task-reg-run-flex-gateway.adoc[tags=app-docker-heading;reg-command-intro;sub-coll-info;app-replace-content;environment-replace-content;replace-content;reg-command-heading;reg-command-1;docker-reg-command;app-reg-command;environment-reg-command;connected-reg-command;organization-reg-command;output-reg-command-docker;reg-command-2;after-reg;after-reg-2;connected-after-reg;start-command;gateway-connected]
 
-// Register and run with a connected app as a kubernetes ingress controller
-include::partial$task-reg-run-flex-gateway.adoc[tags=app-k8s-heading;k8s-reg-command-intro;sub-coll-info;app-replace-content;environment-replace-content;replace-content;reg-command-heading;reg-command-1;docker-reg-command;app-reg-command;environment-reg-command;connected-reg-command;organization-reg-command;output-reg-command-docker;reg-command-2;after-reg;k8s-after-reg;connected-after-reg;k8s-install-helm-chart;gateway-connected;helm-chart-options]
+[[kubernetes]]
+// Register and run with a connected app in a Kubernetes cluster
+//intro
+include::partial$task-reg-run-flex-gateway.adoc[tags=app-k8s-heading;k8s-reg-command-intro;k8s-mode-overview]
+//sub collected info
+include::partial$task-reg-run-flex-gateway.adoc[tags=sub-coll-info;app-replace-content;environment-replace-content;replace-content]
+//reg command
+include::partial$task-reg-run-flex-gateway.adoc[tags=reg-command-heading;reg-command-1;docker-reg-command;app-reg-command;environment-reg-command;connected-reg-command;organization-reg-command;output-reg-command-docker;reg-command-2;after-reg;k8s-after-reg;connected-after-reg]
+//install helm chart
+include::partial$task-reg-run-flex-gateway.adoc[tags=k8s-install-helm-chart;gateway-connected]
+//helm config options
+include::partial$task-reg-run-flex-gateway.adoc[tags=helm-chart-options]
+
+//TODO: ARE THERE OPENSHIFT OPTIONS HERE?
```

Comment:
OpenShift is like a k8s "distro", so everything that applies for k8s applies for OpenShift as well. Openshift users should know this, so I think it's not necessary to specify it

Reply:
mentioned in https://github.com/mulesoft/docs-gateway/pull/219